container/docker/factory.go

```diff
@@ -197,6 +208,80 @@ func startThinPoolWatcher(dockerInfo *dockertypes.Info) (*devicemapper.ThinPoolW
 	return thinPoolWatcher, nil
 }
 
+func getKernelVersion() (string, error) {
```

Comment:
This function already exists here: https://github.com/google/cadvisor/blob/master/machine/info.go#L142